src/media/css/feed/shelves.styl

```diff
@@ -121,3 +121,26 @@ shelf_curve_color($color) {
 [data-shelf-landing-carrier="mtn"] {
     collection_background_colors($mtn_color);
 }
+
+@media $base-tablet {
+    [data-shelf-landing-carrier="telefonica"] {
+        .feed-landing-header {
+            padding: 0;
+        }
+        .feed-landing-curve {
+            border-radius: 0;
+            padding: 40px 0;
+        }
+        .has-desc.feed-landing-header-inner {
+            flex-stack();
+            flex-vertical-center();
```

Comment:
and i'll be getting rid of this mixin too. so we can just use the rules we'd normally use for flex vertical centering